libxxf86vm.yaml

```diff
@@ -0,0 +1,51 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/libxxf86vm/APKBUILD
```

Comment:
this is needed now?

Reply:
It gets generated automatically by melange convert.